apps/api/src/migration/1676405526677-CreateReportView.ts

```diff
@@ -0,0 +1,278 @@
+              asa.status_id = (SELECT ias.id FROM ien_applicant_status ias WHERE ias.status = 'Completed Simulation Lab Assessment (SLA)')   
+          ) as completed_sla,
+          (SELECT min(start_date) FROM public.ien_applicant_status_audit asa
+            WHERE asa.applicant_id = hwa.id AND
+              asa.status_id = (SELECT ias.id FROM ien_applicant_status ias WHERE ias.status = 'Completed NCAS')   
+          ) as completed_ncas,
+          (SELECT min(start_date) FROM public.ien_applicant_status_audit asa
+            WHERE asa.applicant_id = hwa.id AND asa.status_id IN (
+              SELECT ias.id FROM ien_applicant_status ias WHERE ias.category = 'IEN Recruitment Process'
+            )
+          ) as recruitment,
+          (SELECT min(start_date) FROM public.ien_applicant_status_audit asa
+            WHERE asa.applicant_id = hwa.id AND
+              asa.status_id IN (
+                SELECT ias.id FROM ien_applicant_status ias WHERE ias.status IN (
+                  'Candidate Passed Pre-Screen',
+                  'Candidate Did Not Pass Pre-Screen'
+                )
+              )
+          ) as pre_screen,
+          (SELECT min(start_date) FROM public.ien_applicant_status_audit asa
+            WHERE asa.applicant_id = hwa.id AND
+              asa.status_id IN (
+                SELECT ias.id FROM ien_applicant_status ias WHERE ias.status IN (
+                  'Candidate Passed Interview',
+                  'Candidate Did Not Pass Interview'
+                )
+              )
+          ) as interview,
+          (SELECT min(start_date) FROM public.ien_applicant_status_audit asa
+            WHERE asa.applicant_id = hwa.id AND
+              asa.status_id IN (
+                SELECT ias.id FROM ien_applicant_status ias WHERE ias.status IN (
+                  'Candidate Passed Reference Check',
+                  'Candidate Did Not Pass Reference Check'
+                )
+              )
+          ) as reference_check,
+          (SELECT min(start_date) FROM public.ien_applicant_status_audit asa
+            WHERE asa.applicant_id = hwa.id AND
+              asa.status_id IN (
+                SELECT ias.id FROM ien_applicant_status ias WHERE ias.status IN (
+                  'Job Offer Accepted',
+                  'Job Offer Not Accepted',
+                  'Job Competition Cancelled',
+                  'HA is Not Interested',
+                  'No Position Available',
+                  'Candidate Withdrew from Competition'
+                )
+              )
+          ) as competition_outcome,
+          (SELECT min(start_date) FROM public.ien_applicant_status_audit asa
+            WHERE asa.applicant_id = hwa.id AND
+              asa.status_id IN (
+                SELECT ias.id FROM ien_applicant_status ias WHERE ias.category = 'BC PNP Process'
+              )
+          ) as immigration,
+          (SELECT min(start_date) FROM public.ien_applicant_status_audit asa
```

Comment:
Nit - This is pretty hard to read. Would it be possible to add a new line between each select statement?